include/common_structs.h

```diff
@@ -208,7 +212,7 @@ typedef struct Npc {
     /* 0x0C0 */ s8 unk_C0;
     /* 0x0C1 */ s8 paletteCount;
     /* 0x0C2 */ char unk_C2[2];
-    /* 0x0C4 */ s32* spritePaletteList;
+    /* 0x0C4 */ PAL_PTR* spritePaletteList;
```

Comment:
This has gone from s32* to u16** - is the double-indirection correct?

Reply:
Yes. This is a pointer to a list of palette pointers in the sprite data terminated by -1. Not sure how the previous definition was OK, unless it was being cast. Going back to s32 is not OK.